From 52ae9650b3a09306608cc266bd2ca5b4d789d453 Mon Sep 17 00:00:00 2001
From: Johannes Berg <johannes.berg@intel.com>
Date: Thu, 21 Dec 2023 12:51:14 +0100
Subject: [PATCH 25/28] bump version to 6.7

This already has some "will be 6.8" content, but
that's not entirely closed yet.

Change-Id: I16beefdf8b5e37fc72948d8a874772dc8c97e7b2
Signed-off-by: Johannes Berg <johannes.berg@intel.com>
---
 version.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version.sh b/version.sh
index 9895056..3091e1e 100755
--- a/version.sh
+++ b/version.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-VERSION="5.19"
+VERSION="6.7"
 OUT="$1"
 
 # get the absolute path for the OUT file
-- 
2.39.2

